### PR TITLE
bug1491551-contd:Add api to fetch lastfire info by hookGroupId and hookId - initial

### PR DESCRIPTION
**Pull request by Biboswan**
Once this is set. We need to figure how to expire the info

### CODE REVIEW

**Comment by djmitche** [comment]
It's a holiday weekend in the US so I probably won't get to look at this until Wednesday.  But it's on my list!

**Comment by Biboswan** [comment]
ok! happy holiday

File: src/v1.js [hunk 1]
```diff
@@ -556,3 +556,34 @@ const triggerHookCommon = async function({req, res, hook, payload, firedBy}) {
...
+  method:       'get',
+  route:        '/hooks/:hookGroupId/:hookId/listLastFires',
+  name:         'listLastFires',
+  idempotent:   true,
+  output:       'list-lastFires-response.yml',
+  title:        'list lastFires for a given hookId',
+  stability:    'experimental',
+  description: [
+    'This endpoint will return a list of all lastfire definitions for the given `hookGroupId`',
+    'and hookId.',
```
Comment (djmitche) [inline src/v1.js]: Let's re-word this from the user perspective: "This endpoint will return information about the the last few times this hook has been fired, including whether the hook was fired successfully or not."

**Comment by djmitche** [comment]
Hey presto!

https://hooks.taskcluster.net/v1/hooks/project-gecko/in-tree-action-3-generic%2F5c80b07319/last-fires

**Pull request merged**